src/IntervalSets.jl

```diff
@@ -337,6 +337,12 @@ maximum(d::TypedEndpointsInterval{L,:open}) where L = throw(ArgumentError("$d is
 
 extrema(I::TypedEndpointsInterval) = (infimum(I), supremum(I))
 
+minimum(::typeof(abs), I::TypedEndpointsInterval{:closed,:closed}) =
+    minimum(I) < 0 < maximum(I) ?
+        zero(minimum(I)) :
```

Comment:
```suggestion
minimum(::typeof(abs), I::TypedEndpointsInterval{:closed,:closed,T}) where T=
    zero(T) in I ?
        zero(T) :
```
It's a rather obscure point of implementation, but since eg. intervals of DateTime are supported, `zero` might be more correct for some such type.

Apart from that LGTM.